lib/Config/Database.php

```diff
@@ -1,64 +1,23 @@
 <?php
+
 namespace MailPoet\Config;
 
-use ORM as ORM;
-use PDO as PDO;
+use ORM;
+use PDO;
 
 if (!defined('ABSPATH')) exit;
 
-require_once(ABSPATH . 'wp-admin/includes/plugin.php');
-
 class Database {
-  public $driver_option_wait_timeout = 60;
-
-  function init() {
-    $this->setupConnection();
+  function init(PDO $pdo) {
+    ORM::setDb($pdo);
     $this->setupLogging();
-    $this->setupDriverOptions();
     $this->defineTables();
   }
 
-  function setupConnection() {
-    ORM::configure(Env::$db_source_name);
```

Comment:
I think that this was the only usage of Env::$db_source_name so I think we remove it together with method which is used to generates it: https://github.com/mailpoet/mailpoet/blob/b3cdf74fa614914a368289247f3cbcbc9efb1239/lib/Config/Env.php#L86

Reply:
Fixed.